modules/navier_stokes/src/fvkernels/INSFVMomentumAdvection.C

```diff
@@ -167,12 +208,18 @@ INSFVMomentumAdvection::computeResidualsAndAData(const FaceInfo & fi)
     }
     else
     {
-      const auto [interp_coeffs, advected] =
-          interpCoeffsAndAdvected(*_rho_u, advected_face_arg, state);
+      auto [interp_coeffs, advected] = interpCoeffsAndAdvected(
+          _use_norm_for_momentum_limiter ? *_mom_abs : *_rho_u, advected_face_arg, state);
```

Comment:
What would be the justification for this? For example, in the development of the boundary layer, we want MinMod to limit with the velocity perpendicular to the wall and not with normal of the velocity that will be mainly given by streamwise velocity